### ISSUE TITLE
sh: 1: ~/folder/TwitchDownloadCLI permission denied

### ISSUE THREAD

**Issue by Spacemaan13** [lay295/TwitchDownloader#1066]
### Checklist

- [X] I have checked the [issue page](https://github.com/lay295/TwitchDownloader/issues?q=is%3Aissue) for duplicates

### Write stuff here

Hello, kinda new here sorry if this has already been solved and went under my nose. 
I'm also a beginner user on linux. Made the switch from windows not so long ago so i'm not familiar with everything about folders and rights. 

I'm using ubuntu, installed Twitch Downloader and put TwitchDownloadCLI in a folder "Extraits". I'm using the custom TWDCLI binary and put the correct path to it, which is my "Extraits" folder. 

When trying to download chat from a stream it gives me the error " sh: 1: ~/Extraits/TwitchDownloadCLI permission denied". 

Can you help me solve it ? Thanks

**Comment by ScrubN** [comment]
Did you give the binary executable rights with `sudo chmod +x TwitchDownloaderCLI` first? If that doesn't fix it, a capture of the terminal output, including the command, would help give more context.

**Comment by Spacemaan13** [comment]
Fantastic, it worked ! Thanks a lot. 
I opened the terminal from my folder ~/Extraits and used the command. It asked me my password and didn't return anything so i opened Twitch Downloader and tried to dl a chat from a stream. 

Now the logs return the copyrights and other informations.  I tried multiple options before answering you in case something went wrong. Everything is working fine now. 

Surely it was something basic that i missed but now thanks to you i've learned how to give executable rights ! :) 
Thanks again !
